frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -42,6 +42,7 @@
         vm.sortColumn = 'rank';
         vm.reverseSort = false;
         vm.columnIndexSort = 0;
+        vm.submissionsExhausted = true;
```

Comment:
This variable name is confusing. Can you please change it to `disableSubmit`?

Reply:
Done